WBooks/Screens/Library/CustomBookCell/CustomBookCellViewModel.swift

```diff
@@ -15,9 +15,9 @@ class CustomBookCellViewModel {
         self.bookModel = bookModel
     }
     
-    var image: UIImage {
-        guard let imageName = bookModel.image, let image = UIImage(named: imageName) else { return UIImage() }
-        return image
+    var image: String? {
+        guard let imageName = bookModel.image else { return nil }
```

Comment:
you could validate nil and "" in this `guard`. Also there is a `.isEmpty` and in wolmoCore we have `.isNotEmpty`, so you don't need to use `!= nil`